scriptmodules/supplementary/audiosettings.sh

```diff
@@ -89,9 +106,7 @@ function _bcm2835_alsa_compat_audiosettings() {
                 alsactl store
                 ;;
             R)
-                /etc/init.d/alsa-utils reset
-                alsactl store
-                rm -f "$home/.asoundrc"
+                _reset_alsa_audiosettings
                 printMsgs "dialog" "Audio settings reset to defaults"
```

Comment:
`printMsgs` is redundant here, it's included in the `_reset_alsa_audiosettings`.